pages/windows/cmd.md

```diff
@@ -0,0 +1,35 @@
+# cmd
+
+> The Windows command interpreter.
+
+- Start a new instance of the command interpreter:
+
+`cmd`
+
+- Run the specified command and terminate:
```

Comment:
Perhaps ...`and then exit` would read better?

My thinking is that the example below could be altered to says something like `and then enter an interactive shell` or something.

Reply:
Sorry, been away this weekend. :+1: I think that definitely is more clear. :slight_smile: